pkgs/by-name/si/simplemoji/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  autoPatchelfHook,
+  darwin,
+  fetchFromGitHub,
+  lib,
+  libxkbcommon,
+  fontconfig,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+  wayland,
+  xorg,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "simplemoji";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "SergioRibera";
+    repo = "simplemoji";
+    rev = "v${version}";
+    hash = "sha256-MU31gRikMqXelWjwjMPZtfbDpybYlPTFr1u9w2843Kc=";
+  };
+
+  cargoHash = "sha256-NvNNc+V7wloPxM532uNrXoou4vnKLhFK29L2/17eB98=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ] ++ lib.optionals stdenv.hostPlatform.isLinux [ autoPatchelfHook ];
+
+  buildInputs =
+    lib.optionals stdenv.hostPlatform.isDarwin [
+      darwin.apple_sdk.frameworks.Accessibility
+      darwin.apple_sdk.frameworks.AppKit
+      darwin.apple_sdk.frameworks.CoreFoundation
+      darwin.apple_sdk.frameworks.CoreGraphics
+      darwin.apple_sdk.frameworks.Foundation
+      darwin.apple_sdk.frameworks.OpenGL
+    ]
```

Comment:
```suggestion
```

Not needed anymore

Reply:
Oooh, I didn't know it was no longer necessary, thank you!